parlai/core/metrics.py

```diff
@@ -411,26 +444,43 @@ def normalize_answer(s):
     return s
 
 
-def aggregate_named_reports(named_reports: Dict[str, Dict[str, Metric]]):
+def aggregate_named_reports(
+    named_reports: Dict[str, Dict[str, Metric]], micro_average: bool = False
+):
     """
     Aggregate metrics from multiple reports.
 
     :param reports: Dict of tasks -> metrics.
+    :param micro_average: If true, top level metrics will be the macro average.
```

Comment:
nit: `If true, top level metrics will be the _micro_ average`

Reply:
hardly a nit!